app/templates/components/toc-section.hbs

```diff
@@ -15,7 +15,11 @@
     <ul aria-labelledby={{this.id}} class="level-3 list-unstyled">
       {{#each @result.contents as |content index|}}
         <li class={{if index "mt-2"}}>
-          <a class="bg-none" href="#{{id-for-deprecation content.id content.anchor}}" onclick={{this.navigateToLink}}>
+          <a
+            class="bg-none"
+            href="#{{id-for-deprecation content.id content.anchor}}"
+            {{on "click" this.navigateToLink}}
```

Comment:
Note to myself and other maintainers:

It doesn't feel okay to include a click event on an anchor tag, and/or use `later`. Maybe, in the future, we can use a modifier to achieve what the following code does (`navigateToLink` is not the best name)?

https://github.com/ember-learn/deprecation-app/blob/e9cd33a2bdeee82c55aab63ee6e8d96474976a14/app/components/toc-section.js#L33-L42

Reply:
> Note to myself and other maintainers:
> 
> It doesn't feel okay to include a click event on an anchor tag, and/or use `later`. Maybe, in the future, we can use a modifier to achieve what the following code does (`navigateToLink` is not the best name)?
> 
> https://github.com/ember-learn/deprecation-app/blob/e9cd33a2bdeee82c55aab63ee6e8d96474976a14/app/components/toc-section.js#L33-L42

Yeah I know it's not great. Was trying to solve for a specific issue. Keep in mind this is all going to come out in the redesign anyway.

Reply:
Yep, sounds good.